Unreal/CarlaUnreal/Config/DefaultEngine.ini

```
@@ -69,9 +70,6 @@ MaxAgents=1000
 SpatializationPlugin=
 ReverbPlugin=
 OcclusionPlugin=
```

```suggestion
OcclusionPlugin=
-TargetedRHIs=SF_VULKAN_SM5
+TargetedRHIs=SF_VULKAN_SM5
+TargetedRHIs=GLSL_430
```

For now, I will remove this code as we did not get a consensus about it and seems out of the scope of this review.
Let's approve this review and create another to focus on this topic.